### PR TITLE
[GL-4321] Documentation for embedded resources

### PR DESCRIPTION
**Pull request by dse0**
I created a new section "Sharing Externally" and moved `Slack Integration` there as well. (and presumably will have public sharing here too.)

screencapture-localhost-8005-docs-sharing-data-externally-iframe-embedding-2023-04-26-19_12_23

### CODE REVIEW

**Comment by davidsonsarah** [comment]
This looks great, so excited to have this published! 

Only change requested: can you please add in a section at the bottom that says "Coming Soon: Public Link Sharing" and include a link to the product roadmap?

**Comment by dse0** [comment]
> This looks great, so excited to have this published!
> 
> Only change requested: can you please add in a section at the bottom that says "Coming Soon: Public Link Sharing" and include a link to the product roadmap?

Sure, please remind me to remove it when we add a separate page for it.

File: docs/docs/sharing-data-externally/iframe-embedding.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+```
+https://glean.io/app/sv/Sgf5dLgMfFeafeDe
+```
+
+Then your self-service embed URL would be:
+```
+https://glean.io/app/sv/Sgf5dLgMfFeWVzie?viewMode=embed&projectInviteToken=p-Ip2Lw8ep6XafEzSo_NF
+```
+
+Embedded resources are also compatible with [Get Access Domains](../project-management/get-access-domains.md). If enabled, any user with an e-mail address from your organization's domain will be able to create a Glean account via the embedded resource.
```
Comment (glean-calder) [inline docs/docs/sharing-data-externally/iframe-embedding.md:41]: Nit: I think we always write `email` instead of `e-mail` in docs.

**Review by glean-calder — CHANGES_REQUESTED** [review]
Looks great! Just a blocking request on including docs about models.

File: docs/docs/sharing-data-externally/iframe-embedding.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+Glean supports embedding Saved Views and Dashboards in external applications via an iframe.
+
+## Embedding a Resource
+
+To embed a Glean resource, add `viewMode=embed` to the URL parameters of your Saved View or Dashboard URL and set that URL to be the `src` of an `iframe` HTML tag.
```
Comment (glean-calder) [inline docs/docs/sharing-data-externally/iframe-embedding.md:5]: We should also mention that you can embed models, and probably add an example of this too.

**Review by glean-calder — CHANGES_REQUESTED** [review]
Not sure where that comment about models went - rewrote it

File: docs/docs/sharing-data-externally/iframe-embedding.md (hunk 2 above)
Comment (glean-calder) [inline docs/docs/sharing-data-externally/iframe-embedding.md:5]: Side note - we don't seem to consistently use capitalization for Glean resources like `Saved View` and `Dashboard`. This does it the way I think we should - capitalizing them. But other docs don't.

@davidsonsarah, any thoughts on this before I write up a ticket to amend other docs that don't conform?

File: docs/docs/sharing-data-externally/iframe-embedding.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+Glean supports embedding Models, Saved Views, and Dashboards in external applications via an iframe.
+
+## Embedding a Resource
+
+To embed a Glean resource, add `viewMode=embed` to the URL parameters of your Model, Saved View, or Dashboard URL and set that URL to be the `src` of an `iframe` HTML tag.
```
Comment (glean-calder) [inline docs/docs/sharing-data-externally/iframe-embedding.md:5]: Nit: seems like we call these `Data Models` elsewhere in docs more than `Models`? Not blocking at all - we definitely need to take a pass at normalizing Glean vocabulary in our docs (cc @davidsonsarah for visibility)

**Review by glean-calder — APPROVED** [review]
Awesome!

**Comment by farkmarnum** [comment]
@dse0 tiny nit — I believe the convention these days is to not capitalize the F in "iframe":

- [Mozilla](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe)
- [W3C](https://www.w3.org/TR/2011/WD-html5-20110525/the-iframe-element.html#the-iframe-element)

**Pull request merged**